mods/cnc/chrome/settings.yaml

```diff
@@ -451,6 +451,20 @@ Container@SETTINGS_PANEL:
 											X: PARENT_RIGHT-WIDTH+1
 											Width: 80
 											Height: 25
+								Container@SUPPORTPOWER_TEMPLATE:
```

Comment:
These are just duplicating the production template.  Could you reuse that instead?